sonar-project.properties

```diff
@@ -1,7 +1,9 @@
-sonar.projectKey = yoti-web-sdk:python
-sonar.projectName = python-sdk
+sonar.host.url = https://sonarcloud.io
+sonar.organization = getyoti
+sonar.projectKey = getyoti:python
+sonar.projectName = Python SDK
 sonar.projectVersion = 2.10.0
-sonar.exclusions=yoti_python_sdk/tests/**,examples/**
+sonar.exclusions = yoti_python_sdk/tests/**,examples/**
```

Comment:
Should we also be excluding the (protobuf/) `attribute_public_api`, `common_public_api` and `share_public_api` folders?